packages/server/src/login/login.controller.ts

```diff
@@ -19,15 +21,23 @@ import {
   KhouryRedirectResponse,
   KhouryDataParams,
   KhouryStudentCourse,
-  KhouryTACourse
+  KhouryTACourse,
 } from '@template/common';
 import { NonProductionGuard } from '../../src/non-production.guard';
 import { ConfigService } from '@nestjs/config';
 import { LoginCourseService } from './login-course.service';
 import { CourseSectionMappingModel } from './course-section-mapping.entity';
+import * as httpSignature from 'http-signature';
+import * as path from 'path';
+import * as fs from 'fs';
 
 @Controller()
 export class LoginController {
+  PUBLIC_KEY = fs.readFileSync(
```

Comment:
i think cypress is erroring on this line just fyi

Reply:
thanks for the heads up!